db/models.py

```diff
@@ -14,3 +14,44 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(null=True, blank=True)
+    actors = models.ManyToManyField(Actor, related_name="movie_actors_id")
```

Comment:
It should be a simple name that represents movies on models of actors or genres. How do you think would be easier to access it:
```
actor.movie_actors_id
OR
actor.movies

Reply:
Please change related name to more relevant